### DISCUSSION TITLE
Bulk Emails

### DISCUSSION THREAD

**Discussion by HazemMohamed252** [statamic/cms#5605] [Q&A]
How to include feature that enable me to send Bulk Emails for multi users ? Are there any way to do this please as i want it urgent

**Comment by edalzell** [comment]
What's your use case? It's not a feature in Statamic, at the moment.

**Reply by HazemMohamed252** [reply]
I know it is not statamic feature but i want to do something enable me to send bulk emails for multi users. Any way ?

**Reply by edalzell** [reply]
Depending on your use case, you might be able to make a bulk action?

Otherwise you could create a command, or even your own CP view.

**Reply by HazemMohamed252** [reply]
Sorry can you explain more. I will explain again, assume we have a link on the sidebar call : Bulk Emails, when i click on it it show page with dropdown list with multi select and a textarea to write any text then send email by click on button

**Comment by edalzell** [comment]
Sure make your own CP page, you can do whatever you want on there.

**Reply by HazemMohamed252** [reply]
I am sorry but can you support me with few steps to do that, how to make my own CP page ?

**Reply by edalzell** [reply]
take a look at some addons, for example Akismet, it makes a CP page. Or Runway.
